Added T4 template path to Advanced dialog box.

ClaudioBiagioli opened the pull request:
Also added some missing italian translations.

ErikEJ commented:
Wonderful! 

Can you share a screenshot?

Did you also check the italian .vsct file (ButtonText property) ?

ClaudioBiagioli commented:
Hi @ErikEJ I added the the T4 path field to the Advanced Dialog box, like in the screen shot below.

![Screenshot 2024-01-08 143431](https://github.com/ErikEJ/EFCorePowerTools/assets/56717465/a0b7d7c5-432b-4bd0-8f63-14ac6b5233e0)


I've also checked the vsct file and added some translations there too.

C.

ErikEJ commented:
Thanks for a great contribution!
